specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -758,11 +920,7 @@
             "$ref": "./network.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "name": "location",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The location of the where the association is present."
+            "$ref": "#/parameters/LocationParameter"
```

Comment:
>#/parameters/LocationParameter [](http://example.com/codeflow?start=21&length=30)

Consider using LocationParameter from v3 common types. This is a suggestion, it won't block ARM signoff.